Source/UI/Controllers/IssuersScreen/MLCardFormIssuerTableViewCell.swift

```diff
@@ -41,11 +44,35 @@ final class MLCardFormIssuerTableViewCell: UITableViewCell {
 extension MLCardFormIssuerTableViewCell {
     func setupCell(with issuer: MLCardFormIssuer, radioButtonOn: Bool) {
         radioButtonOn ? setupRadioButton(radioButtonOn: true) : setupRadioButton(radioButtonOn: false)
-        if let imageUrl = issuer.imageUrl {
-            issuerImageView.setRemoteImage(imageUrl: imageUrl)
-        }
+        checkHasImage(issuer: issuer)
         accessibilityLabel = issuer.name
     }
+    
+    private func checkHasImage(issuer: MLCardFormIssuer) {
+        guard let imageURL = issuer.imageUrl else {
+            titleLabel.text = issuer.name
+            issuerImageView.isHidden = true
+            titleLabel.isHidden = false
+            return
+        }
+        
+        DispatchQueue.global(qos: .background).async {
+            do {
+                let data = try Data.init(contentsOf: URL.init(string: issuer.name)!)
+                    DispatchQueue.main.async {
+                        let image: UIImage = UIImage(data: data)!
```

Comment:
Here it can be like this:
`let image = UIImage(data: data)` (Removing force unwrap)

Reply:
well noticed, I agree!